app/Http/Controllers/DepartureReleaseController.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Exceptions\Release\Departure\DepartureReleaseDecisionNotAllowedException;
+use App\Models\Controller\ControllerPosition;
+use App\Models\Release\Departure\DepartureReleaseRequest;
+use App\Rules\Controller\ControllerPositionValid;
+use App\Services\DepartureReleaseService;
+use Carbon\Carbon;
+use Carbon\CarbonImmutable;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Log;
+
+class DepartureReleaseController
+{
+    private DepartureReleaseService $departureReleaseService;
+
+    public function __construct(DepartureReleaseService $departureReleaseService)
+    {
+        $this->departureReleaseService = $departureReleaseService;
+    }
+
+    public function makeReleaseRequest(Request $request): JsonResponse
+    {
+        $validated = $request->validate(
+            [
+                'callsign' => 'required|string',
+                'requesting_controller_id' => [
+                    'required',
+                    'integer',
+                    function ($attribute, $value, $fail) {
+                        if (!ControllerPosition::where('id', $value)->canRequestDepartureReleases()->exists()) {
+                            $fail(sprintf('Controller position %d cannot request departure releases', $value));
+                        }
+                    },
+                    'not_in:target_controller_ids.*',
+                ],
+                'target_controller_id' => [
+                    'required',
+                    'integer',
+                    function ($attribute, $value, $fail) {
```

Comment:
Can this be its own rule object perhaps?

Reply:
Or maybe even middleware for this request?

Reply:
Done!